properties/handlers.py

```diff
@@ -2,34 +2,37 @@
 from six import string_types
 
 
-def _set_listener(instance, observer):
-    """
-        Add listeners to a Properties class instance
-    """
+def _set_listener(instance, obs):
+    """Add listeners to a Properties class instance"""
 
-    for name in observer.names:
+    for name in obs.names:
         if name not in instance._listeners:
-            # TODO: extend to different kinds of observers
-            # instance._listeners[name] = {observer.kind: [observer]}
-            instance._listeners[name] = [observer]
-        else:
-            instance._listeners[name] += [observer]
+            instance._listeners[name] = {'validate': [], 'observe': []}
+        instance._listeners[name][obs.mode] += [obs]
```

Comment:
I was thinking that this might be `observe.always`  or `observe.change` etc. rather than validator. We can add that functionality later if it becomes necessary.